src/functions-and-arrays.js

```diff
@@ -1,19 +1,65 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(number1, number2){
+  if (number1 < number2) {
+    return number2;
+    
+  } else {
+    return number1;
+  }
+}
+
+console.log(maxOfTwoNumbers(3,4));
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(array){
```

Comment:
You should check if the array is empty, if it is, Jasmine asks you to return null. Something like this:

if (!array.length) return null;

or like this:

if (array.length === 0) {
    return null;
}